src/RedirectExtension.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Heyday\SilverStripeRedirects\Source;
+
+use SilverStripe\ORM\DataExtension;
+
+class RedirectExtension extends DataExtension
+{
+    public function onAfterDelete()
+    {
+        $redirects = RedirectUrl::get()->filterAny([
+            'FromID' => $this->owner->ID,
```

Comment:
I'm not sure if this should be deleting redirects away from the page.
One of the uses of redirects is to start redirecting people away from a page before you delete it (or while you're replacing it).
Maybe instead it could change the redirect so if it's redirecting away from this page by ID, it instead redirects away from this page by URL?